bot/code_review_bot/tasks/coverity.py

```diff
@@ -48,7 +48,7 @@ class CoverityIssue(Issue):
     def __init__(self, revision, issue, file_path):
         self.revision = revision
         self.path = file_path
-        self.reliability = Reliability(issue['reliability'])
+        self.reliability = Reliability(issue['reliability']) if 'reliability' in issue else Reliability.Unknown
```

Comment:
Don't we want the reliability for NSS too? It seems like very useful information!

Reply:
@abpostelnicu ?

Reply:
That would involve a patch on nss

Reply:
Agreed with @abpostelnicu to use `Reliability.Unknown` on NSS.